search/search_repository.go

```diff
@@ -139,7 +139,9 @@ func trimProtocolFromURLString(urlString string) string {
 }
 
 func escapeCharFromURLString(urlString string) string {
-	return strings.Replace(urlString, ":", "\\:", -1)
+	// Replacer will escape `:` and `)` `(`.
+	var replacer = strings.NewReplacer(":", "\\:", "(", "\\(", ")", "\\)")
```

Comment:
As a general rule; it's always easier to test for what you allow instead of trying to test for everything you don't.